git/examples/git-cli/pipeline.yaml

```
@@ -0,0 +1,63 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: git-pipeline
+spec:
+  workspaces:
+    - name: shared-workspace
+    - name: input
+  tasks:
+    - name: fetch-repository
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+      params:
+        - name: url
+          value: https://github.com/kelseyhightower/nocode
+        - name: subdirectory
+          value: ""
+        - name: deleteExisting
+          value: "true"
+    - name: git-cli
+      taskRef:
+        name: git-cli
+      runAfter:
+        - fetch-repository
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+        - name: input
+          workspace: input
+      params:
+        - name: GIT_USER_NAME
+          value: $(username)
```

`$(username)` is confusing here and indicates that the value filled somewhere and you are using here

So can we change something like below 
```suggestion
          value: username
```
OR
```suggestion
          value: username-name
```

---

We can, thanks for the suggestion :)